### PR TITLE
 [luci] Verify quantization of Sqrt

### PR DESCRIPTION
**Pull request by dayo09**
This commit verifies quantization of sqrt.

ONE-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

----

For #6367

### CODE REVIEW

**Comment by dayo09** [comment]
Rebased, ready to be merged :-D

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 1]
```diff
@@ -992,6 +1011,27 @@ TEST(QuantizedModelVerifierTest, FloorDiv_wrong_granularity_NEG)
   TEST_WITH_WRONG_GRANULARITY_TARGET(FloorDivTestGraph, Type::S16, Granularity::ChannelWise, g.y());
 }
 
+TEST(QuantizedModelVerifierTest, Sqrt)
```
Comment (dayo09) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:1014]: I will put this below rsqrt (don't know why it has reversed)

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 2]
```diff
@@ -530,6 +530,44 @@ class RsqrtTestGraph final : public luci::test::TestIOGraph
...
+    }
+    output()->from(_elu);
+
+    set_minmax_to_non_const(g(), -1, 1);
+  }
+
+public:
+  luci::CircleElu *_elu = nullptr;
+};
+
```
Comment (jinevening) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:570]: Maybe accidentally included?

Comment (dayo09) [reply compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:570]: OMG

**Comment by dayo09** [comment]
@jinevening Resolved :-D

File: compiler/luci/pass/src/QuantizedModelVerifier.test.cpp [hunk 3]
```diff
@@ -1013,6 +1032,27 @@ TEST(QuantizedModelVerifierTest, Rsqrt_wrong_granularity_NEG)
   TEST_WITH_WRONG_GRANULARITY(RsqrtTestGraph, Type::S16, Granularity::ChannelWise);
 }
 
+TEST(QuantizedModelVerifierTest, Sqrt)
+{
+  TEST_WITH_GRAPH(SqrtTestGraph, Type::U8, Granularity::LayerWise);
+  TEST_WITH_GRAPH(SqrtTestGraph, Type::U8, Granularity::ChannelWise);
+  TEST_WITH_GRAPH(SqrtTestGraph, Type::S16, Granularity::ChannelWise);
+}
```
Comment (jinevening) [inline compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:1040]:
```suggestion
  SUCCEED();
}
```

Comment (jinevening) [reply compiler/luci/pass/src/QuantizedModelVerifier.test.cpp:1040]: Lets' add `SUCCEED();` from now on.

PTAL https://github.com/Samsung/ONE/issues/6571

**Review by jinevening — APPROVED** [review]
LGTM Thank you.

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**